girder_utils/rest.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework.pagination import LimitOffsetPagination
```

Comment:
I might want to add an optional dependency on DRF here. I'll look at how some other libraries do it.

Reply:
Yeah I wasn't sure on that. This package as a whole doesn't depend on it, and I assumed anyone importing this specific utility would know to have it installed. I don't care either way.

Reply:
I think that's fine for now.

Reply:
I want to rename this new module to `rest_framework` (i.e. a file of `rest_framwork.py`) though, which follows the way that a number of other third-party Django libraries provide optional DRF support.